app/uk/gov/hmrc/apiplatformevents/controllers/ApplicationEventsController.scala

```diff
@@ -62,6 +60,22 @@ class ApplicationEventsController @Inject()(val env: Environment,
     }
   }
 
+  def teamMemberRemoved() = Action.async(playBodyParsers.json) { implicit request =>
+    withJsonBody[TeamMemberRemovedEvent]{ event=>
+      service.captureTeamMemberRemovedEvent(event) map {
+        case true => {
+          Created
```

Comment:
Some common code here we can probably refactor into one place. One for the map, another for the recover